php/addtorrent.php

```diff
@@ -31,6 +31,9 @@
 		if((strlen($dir_edit)>0) && !rTorrentSettings::get()->correctDirectory($dir_edit))
 			$uploaded_files = array( array( 'status' => "FailedDirectory" ) );
 	}
+	$addition = array();
```

Comment:
Please initialize this as `null` to match the function prototype. We can sneak this into ruTorrent v4.0 stable if completed.

`static public function sendTorrent($fname, $isStart, $isAddPath, $directory, $label, $saveTorrent, $isFast, $isNew = true, $addition = null)`